### PR TITLE
fix(challenges): fix grammar and spelling errors

### PR DESCRIPTION
**Pull request by sdabhi23**
fix grammar and spelling errors in `Get Route Parameter Input from Client` challenge

#### Description
fix grammar and spelling errors in `Get Route Parameter Input from Client` challenge

#### Pre-Submission Checklist

- [x] Your pull request targets the `dev` branch.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/challenge-tests`)
- [x] All new and existing tests pass the command `npm test`.
- [x] Use `npm run commit` to generate a conventional commit message.
    Learn more here: <https://conventionalcommits.org/#why-use-conventional-commits>
- [x] The changes were done locally on your machine and NOT GitHub web interface.
    If they were done on the web interface you have ensured that you are creating conventional commit messages.

#### Checklist:

- [x] Tested changes locally.
- [x] Addressed currently open issue (replace XXXXX with an issue no in next line)

Closes [freeCodeCamp #17991](https://github.com/freeCodeCamp/freeCodeCamp/issues/17991)

### CODE REVIEW

File: challenges/05-apis-and-microservices/basic-node-and-express.json [hunk 1]
```diff
@@ -188,20 +175,18 @@
...
       "description": [
-        "When building an API, we have to allow users to comunicate us what they want to get from our service. For example, if the client is requesting information about a user stored in the database, they need a way to let us know which user they're interested in. One possible way to achieve this result is using route parameters. Route parameters are named segments of the URL, delimited by slashes (/). Each segment captures the value of the part of the URL which matches its position. The captured values can be found in the <code>req.params</code> object.",
+        "When building an API, we have to allow users to communicate to us what they want to get from our service. For example, if the client is requesting information about a user stored in the database, they need a way to let us know which user they're interested in. One possible way to achieve this result is using route parameters. Route parameters are named segments of the URL, delimited by slashes (/). Each segment captures the value of the part of the URL which matches its position. The captured values can be found in the <code>req.params</code> object.",
```
Comment (Nirajn2311) [inline challenges/05-apis-and-microservices/basic-node-and-express.json]: > One possible way to achieve this result is using route parameters.

The above sentence should be changed to 

> One possible way to achieve this result is by using route parameters.

Comment (sdabhi23) [reply challenges/05-apis-and-microservices/basic-node-and-express.json]: @Nirajn2311 it's done

**Comment by scissorsneedfoodtoo** [comment]
@sdabhi23, thank you for this fix! Everything LGTM :+1: :+1:

Also, thank you @Nirajn2311 for helping to QA this.

**Pull request merged**